inference-engine/src/vpu/myriad_plugin/myriad_executable_network.h

```diff
@@ -109,9 +122,11 @@ class ExecutableNetwork : public ie::ExecutableNetworkThreadSafeDefault {
     DevicePtr _device;
     GraphMetaInfo _graphMetaData;
     MyriadConfig _config;
+    bool _notConstNet = true;
```

Comment:
I don't think this is a good naming.
A construction like `if (!_notConstNet)` looks too ugly in this case. It's better to call it `isNetworkConstant` and reverse its usage.
I also suggest to leave some comment there on why the behavior is different for constant network case.

Reply:
I've changed it, can you please take a look?